device-types/Cisco/MDS-C9148-16P.yaml

```diff
@@ -0,0 +1,137 @@
+---
+manufacturer: Cisco
+model: MDS 9148 16P
+slug: ds-9148-16p
+part_number: DS-C9148-16P-K9
+u_height: 1
+is_full_depth: true
+airflow: front-to-rear
+weight:
+  - value: 22.0
+    unit: lb
+comments: 'Cisco MDS 9148 Multilayer Fabric Switch'
+console-ports:
+  - name: Console
+    type: rj-45
+module-bays:
+  - name: Slot 1
+    position: '1'
+    description: 'Supervisor Module'
+  - name: Slot 2
+    position: '2'
+    description: 'MDS 9148 FC (1 Slot) Chassis Power Supply'
+  - name: Slot 3
+    position: '3'
+    description: 'MDS 9148 FC (1 Slot) Chassis Power Supply'
+  - name: Slot 4
+    position: '4'
+    description: 'MDS 9148 FC (1 Slot) Chassis Fan Module'
+  - name: Slot 5
+    position: '5'
+    description: 'MDS 9148 FC (1 Slot) Chassis Fan Module'
+  - name: Slot 6
+    position: '6'
+    description: 'MDS 9148 FC (1 Slot) Chassis Fan Module'
+  - name: Slot 7
+    position: '7'
+    description: 'MDS 9148 FC (1 Slot) Chassis Fan Module'
+interfaces:
+  - name: mgmt0
+    type: 1000base-t
+    mgmt_only: true
+  - name: fc{module}/1
```

Comment:
Any reason these have `{module}` in them? This is a device definition file, not a module definition file. If these are not modular ports then they should have the actual value, i.e. `fc1/1`. Also, if you run a `show config` or other applicable command, this should read exactly as the devices config does. On a 9200L for instance, even though you can configure a port by using `int gi1/0/1` it should be typed in this definition file as `GigabitEthernet1/0/1`. I have never used one of these devices so I dont know, just want to mention that in case it applies here.

Reply:
I have removed {module} and replaced it with the numerical value as instructed.